default.nix

```diff
@@ -31,6 +31,7 @@ let
   inherit (pkgs) lib haskell-nix;
   inherit (plutus) haskell agdaPackages;
   inherit (plutus) easyPS sphinxcontrib-haddock;
+  noCross = x: if crossSystem == null then x else {};
```

Comment:
Isn't this handled by the guards in `ci.nix` already?

Reply:
switched to using `meta.platforms`